octobot/logger.py

```diff
@@ -157,6 +158,8 @@ async def init_exchange_chan_logger(exchange_id):
 
 
 async def init_evaluator_chan_logger(matrix_id: str):
+    if os.getenv(constants.ENV_ENABLE_DEBUG_LOGS, False):
+        return
```

Comment:
Maybe we should let orders / position / liquidation / trades channels

Reply:
I don't know. Orders are still logging 
![image](https://user-images.githubusercontent.com/10454670/129273408-8e12efe2-a426-4077-8287-46d183ed6aea.png)

I don't think the channel subscription is the right thing to keep.